src/services/DfusionService.ts

```diff
@@ -298,6 +356,7 @@ export class DfusionRepoImpl implements DfusionService {
       })
       .on('error', (error: Error) => {
         params.onError(error)
+        this.handleSubscriptionError(error, { subscription, name: subscriptionName })
```

Comment:
Do we resubscribe right away? shouldn't we have delays?

Reply:
EDIT: I see you did indeed implement a delay. Then my question is, wouldn't this generate a lot of noise in case for example the service is down for 2min --> 120 attempts

Reply:
This whole construct triggers only when `connection is established, but  eth_subscribe doesn't work`
So it's not connect/reconnect/subscribe looping over
There are some delays internally inside web3 already
And most importantly, if server is down, reconnection happens thanks to `autoconnect` option supplied to WebsocketProvider constructor

Reply:
okay, right, there was the autoconnect logic from before
Ok then 👌